.gitignore

```diff
@@ -12,7 +12,7 @@
 /build
 
 # misc
-.vscode/
+# .vscode/
```

Comment:
No sir, the .vscode file contains the settings that are required for eslint and prettier to work. Along with that it also has recommendations for extensions such as eslint, prettier just in case anyone does not have them installed. Also even if anyone has installed eslint and prettier, it probably wont work for typescript files. We need to configure them to track ts files. So this handles everything.